### PR TITLE
update REDIRECT_SIGNOUT based on env

### PR DESCRIPTION
**Pull request by keeysnc**
### Description
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2672

Adding redirects for cognito redirect sign out based on environment.

prod: https://idm.cms.gov/app/UserHome
val: https://impl.home.idm.cms.gov/app/UserHome 
test: https://test.home.idm.cms.gov/app/UserHome 

### Related ticket(s)

https://qmacbis.atlassian.net/browse/MDCT-2638
MDCT-2638

---
### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Comment by keeysnc** [comment]
I manually tested if the correct URL gets populated into `REDIRECT_SIGNOUT` on login but I'm open to suggestions if anyone has another way to test this.

File: services/ui-src/src/config.ts [hunk 1]
```diff
@@ -7,6 +7,21 @@ export const isDevEnv = () => {
   );
 };
 
+let branchName = window._env_.BRANCH_NAME;
+let signOutRedirectURL = window._env_.COGNITO_REDIRECT_SIGNOUT;
+
+const redirectSignOurURL = () => {
+  if (branchName === "master") {
+    return (signOutRedirectURL = "https://test.home.idm.cms.gov/app/UserHome");
```
Comment (BearHanded) [inline services/ui-src/src/config.ts:15]: nit:
I think this is perfectly functional, and works, but returning an assignment is a little more complex than it needs to be for readability. Returning the string itself it probably fine, unless I am missing a reason for needing to reassign those default values.

File: services/ui-src/src/config.ts [hunk 2]
```diff
@@ -28,8 +43,8 @@ const configToExport = {
     APP_CLIENT_ID: window._env_.COGNITO_USER_POOL_CLIENT_ID,
     APP_CLIENT_DOMAIN: window._env_.COGNITO_USER_POOL_CLIENT_DOMAIN,
     IDENTITY_POOL_ID: window._env_.COGNITO_IDENTITY_POOL_ID,
-    REDIRECT_SIGNIN: window._env_.COGNITO_REDIRECT_SIGNIN,
-    REDIRECT_SIGNOUT: window._env_.COGNITO_REDIRECT_SIGNOUT,
+    REDIRECT_SIGNIN: window._env_.COGNITO_REDIRECT_SIGNOUT,
```
Comment (BearHanded) [inline services/ui-src/src/config.ts:46]: REDIRECT_SIGNIN shouldn't need to change, should it?

File: services/ui-src/src/config.ts (hunk 1 above)
Comment (keeysnc) [reply services/ui-src/src/config.ts:15]: I agree, no reason except the dot notation looked a little repetitive but I can update yea I don't think its needed either.

**Review by BearHanded — APPROVED** [review]

**Comment by keeysnc** [comment]
Hey @BearHanded I added the conditional back. The config.cognito.REDIRECT_SIGNOUT still needed to be assigned the correct url.

**Review by britt-mo — APPROVED** [review]

**Review by BearHanded — APPROVED** [review]

**Pull request merged**